test/e2e_test_insecure_oci_registry.sh

```diff
@@ -0,0 +1,49 @@
+#!/usr/bin/env bash
+#
+# Copyright 2021 The Sigstore Authors.
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+set -ex
+
+go build -o cosign ./cmd/cosign
+tmp=$(mktemp -d)
+cp cosign $tmp/
+
+INSECURE_REGISTRY_NAME=${INSECURE_REGISTRY_NAME:-insecure-registry.notlocal}
+INSECURE_REGISTRY_PORT=${INSECURE_REGISTRY_PORT:-5001}
+
+pushd $tmp
+
+pass="$RANDOM"
+export COSIGN_PASSWORD=$pass
+export COSIGN_YES="true"
+export COSIGN_OCI_EXPERIMENTAL=1
+export COSIGN_EXPERIMENTAL=1
```

Comment:
this flag does not exist anymore, you can drop

Reply:
Fixed.

Reply:
still here :)

Reply:
Just to be clear, this env var still exists because ...

`
$ ./cosign sign --allow-insecure-registry --registry-referrers-mode=oci-1-1

Error: invalid argument "oci-1-1" for "--registry-referrers-mode" flag: in order to use  mode "oci-1-1", you must set COSIGN_EXPERIMENTAL=1
main.go:74: error during command execution: invalid argument "oci-1-1" for "--registry-referrers-mode" flag: in order to use  mode "oci-1-1", you must set COSIGN_EXPERIMENTAL=1
`

Reply:
humm, interesting, this should be all removed. ok, thanks!